schemas/TransferTemplate.json

```diff
@@ -65,6 +61,21 @@
       "minItems": 1,
       "uniqueItems": true,
       "items": { "$ref": "FundsTemplate.json" }
+    },
+    "additional_info": {
+      "description": "Includes part_of_payment, case, etc.",
+      "type": "object",
+      "properties": {
+        "part_of_payment": {
```

Comment:
Then why not just remove "part_of_payment" altogether and allow anyone to attach anything to `additional_info`?